projects/bootloader/src/dispatcher.c

```diff
@@ -0,0 +1,61 @@
+#include "dispatcher.h"
+
+#include <stdint.h>
+#include <string.h>
+
+#include "bootloader_can.h"
+#include "bootloader_datagram_defs.h"
+#include "can_datagram.h"
+
+#define MAX_DEST_NODES_SIZE_BYTES 255
+#define MAX_DATA_SIZE_BYTES 2048
+
+// callbacks and context
+static DispatcherCallback s_callback_map[NUM_BOOTLOADER_DATAGRAMS] = { NULL };
+static void *s_context_map[NUM_BOOTLOADER_DATAGRAMS] = { NULL };
+
+// setup datagram rx config
+static void prv_dispatch(void);
+static uint8_t s_destination_nodes[MAX_DEST_NODES_SIZE_BYTES] = { 0 };
+static uint8_t s_data[MAX_DATA_SIZE_BYTES] = { 0 };
+static CanDatagramRxConfig datagram_rx = {
```

Comment:
Nit: should be prefixed with s_ to be inline with our coding standards

Reply:
will be fixed in the next commit